src/api/article/data/controller.js

```diff
@@ -0,0 +1,31 @@
+import { checkIsFound } from 'utils/validation';
+import { sendJson } from 'utils/api';
+
+import Article from 'api/article/article.model';
+
+import ArticleData from './model';
+
+export const create = async ({ params: { articleId }, body }, res, next) => {
+  let article;
+  try {
+    article = await Article.findOne({ _id: articleId }).exec();
```

Comment:
please, use promises here

Reply:
you mean instead of try-catch? I have no idea how to implement that. Can you help?